projects/1.18/assets/terraincognita/terraincognita/lang/zh_cn.json

```
@@ -148,7 +171,7 @@
   "block.terraincognita.hazel_pressure_plate": "榛木压力板",
   "block.terraincognita.hazel_chest": "榛木箱子",
   "block.terraincognita.hazel_trapped_chest": "榛木陷阱箱",
-  "block.terraincognita.vertical_hazel_planks": "竖纹榛木木板",
+  "block.terraincognita.vertical_hazel_planks": "竖纹榛木板",
```

```suggestion
  "block.terraincognita.vertical_hazel_planks": "竖纹榛木木板",
```
采用橡木木板翻译方法，具体相关规则可以去看群里的精华消息，欢迎提出意见